### ISSUE TITLE
Cannot use floating point numbers for patching

### ISSUE THREAD

**Issue by jerlam06** [Qwokka/Cetus#32]
Hello,

I noticed that every time I use a floating point number in my patches, it gets transformed into an integer.
I added this line: `f32.const 0.2`, then saved the patch. When I open it again the line is changed to: `f32.const 0`.
Why is that?

**Comment by Qwokka** [comment]
Hi @jerlam06 

This should be fixed with 5b2e4bd0bf15555123580a37817eee421f11cd1d. Let me know if you have any issues. Thanks!

Jack

**Comment by jerlam06** [comment]
Hi @Qwokka
Thanks for your quick fix, I cloned your branch and unfortunately I get an error:
```
Uncaught (in promise) ReferenceError: WailParser is not defined
    at WebAssembly.webAssemblyInstantiateStreamingHook [as instantiateStreaming] (init.js:895:18)
```
